### DISCUSSION TITLE
which version is stable to migrate from cocos2dx 3.17.2 to axmol ? Should I go for axmol-1.0.1p1 ? I tried v2 before but wasn't able to run the empty project.

### DISCUSSION THREAD

**Discussion by meehjeevan** [axmolengine/axmol#1417] [General]
same as the title

**Comment by rh101** [comment]
@meehjeevan The current dev branch (v2.x) seems to be pretty solid.  Just from personal experience, I ported a very large project from Cocos2d-x v3/v4 to Axmol 1.0, and recently updated it to work with Axmol 2.x.  If you're going to port over from Cocos2d-x, it's probably best you do the work to get it onto Axmol 2.x, saving yourself some time.

Also, what do you mean that you weren't able to run an empty project? ~~Do you have a link to the issue or discussion you created about that?  If you didn't bring it up before, then just explain every single step you took to create the project and to build it in order for us to help you get up and running.~~ Never mind! Found your post [here](https://github.com/axmolengine/axmol/discussions/1354), and can see that you never tried to use the latest version of Axmol 2 with the issue fixed.  Stating that you couldn't run an empty project, when you haven't tried it for over a month, is a little misleading, and just serves to confuse things for anyone reading this.

So, try to build an empty project to verify that the issue is fixed, and if you do run into any problems, then ask for help.
